src/pages/RepoPage/FailedTestsTab/GitHubActions/FrameworkTabs/FrameworkTabs.tsx

```diff
@@ -27,27 +27,22 @@ export function FrameworkTabs() {
 
   return (
     <div>
-      <div className="flex justify-between">
-        <div className="flex gap-1">
-          {Object.keys(FrameworkCopy).map((f) => (
-            <button
-              key={f}
-              className={cs(
-                'px-4 py-2',
-                selectedFramework === f && 'border-b-2 border-ds-gray-octonary'
-              )}
-              onClick={() => setSelectedFramework(f as FrameworkType)}
-            >
-              {f}
-            </button>
-          ))}
-        </div>
+      <div className="flex gap-1">
+        {Object.values(Frameworks).map((framework) => (
```

Comment:
Using `Object.values` gives us a union type of the keys being used in the `FrameworkCopy` const, so that we don't need to cast the type when we set the state.